src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -3017,8 +3016,10 @@ public async Task DynamicTableSizeUpdate_Exceeds_Settings_Throws()
                     using HttpClient client = CreateHttpClient();
                     Exception e = await Assert.ThrowsAsync<HttpRequestException>(() => client.GetAsync(uri));
 
-                    Assert.NotNull(e.InnerException);
-                    Assert.Contains("Dynamic table size update", e.InnerException.Message);
+                    IOException io = Assert.IsType<IOException>(e.InnerException);
+
+                    Assert.NotNull(io.InnerException);
+                    Assert.Contains("Dynamic table size update", io.InnerException.Message);
```

Comment:
We generally avoid checking exception messages like this... when we eventually start localizing, this will be debt that'll need to be cleaned up.

Reply:
I'll take this out then.